src/projects/EnsureThat/EnsureArg.Enums.cs

```diff
@@ -0,0 +1,18 @@
+﻿using JetBrains.Annotations;
+
+namespace EnsureThat
+{
+    using System;
+
+    public static partial class EnsureArg
+    {
+        /* Possible alternative names:
+         * - [ ] IsValidEnum (we have the type constraint, but it's new as of C# 7.3 so reiterating the type in the name might not be bad
```

Comment:
On @danielwertheim

Reply:
I will add xmldoc comments to the methods after you vote (since if you choose IsDefined I'll reference the Enum method directly)

Reply:
I've added input below in the test, where all names are used in one place.